src/tools/yarprobotstatepublisher/src/main.cpp

```diff
@@ -35,11 +35,13 @@ int main(int argc, char *argv[])
     if (rf.check("help"))
     {
         cout<<"Options"<<endl;
-        cout<<"\t--name-prefix            <name-prefix>: prefix of the yarprobotstatepublisher ports (default no prefix)"<<endl;
-        cout<<"\t--tf-prefix              <tf-prefix>: prefix of the published TFs (default no prefix)"<<endl;
-        cout<<"\t--model                  <file-name>: file name of the model to load at startup"<<endl;
-        cout<<"\t--base-frame             <frame-name>: specify the base frame of the published tf tree"<<endl;
-        cout<<"\t--jointstates-topic      <topic-name>: source ROS topic that streams the joint state (default /joint_states)"<<endl;
+        cout<<"\t--name-prefix            <name-prefix> : prefix of the yarprobotstatepublisher ports (default no prefix)"<<endl;
+        cout<<"\t--tf-prefix              <tf-prefix>   : prefix of the published TFs (default no prefix)"<<endl;
+        cout<<"\t--model                  <file-name>   : file name of the model to load at startup"<<endl;
+        cout<<"\t--reduced-model          <boolean-flag>: set true to stream only the link TFs\n"
```

Comment:
Just to understand, you have any idea why all the lines changed? Furthermore, for consistency is nice if you terminate all lines with `endl`, not with `\n` just in one case.

Reply:
> Just to understand, you have any idea why all the lines changed?

Just reordered to be aligned at `:`

> Furthermore, for consistency is nice if you terminate all lines with `endl`, not with `\n` just in one case.

I use `\n` to be consistent with the `cout` usage only for the parameters but not the debug info of the parameter, If you want I can change it. The output with the current format looks 

![Selection_001](https://user-images.githubusercontent.com/6505998/62301476-ffab4380-b478-11e9-8177-40df1eacdbd4.png)